e2eshark/run.py

```diff
@@ -1307,28 +1307,26 @@ def main():
 
     args = parser.parse_args()
     cache_dir = args.cachedir
-    if not os.path.exists(cache_dir):
-        print(f"ERROR: The Cache directory {cache_dir} does not exist")
-        sys.exit(1)
 
-    cache_path = os.path.expanduser(cache_dir)
-    cache_path = os.path.abspath(cache_dir)
 
-    if not os.path.exists(cache_path):
-        print(f"ERROR: The Cache directory {cache_path} does not exist.")
+    cache_dir = os.path.expanduser(cache_dir)
+    cache_dir = os.path.abspath(cache_dir)
+
+    if not os.path.exists(cache_dir):
+        print(f"ERROR: The Cache directory {cache_dir} does not exist.")
         sys.exit(1)
```

Comment:
Curious - why error and exit instead of creating the directory? Is the cache directory expected to be populated with some files at this point?

Reply:
i didn't do this part - - just moved it around a little. might put in another pr to fix this latr